### PR TITLE
Check if webbrowser._tryorder is not None for python 3

### PR DESCRIPTION
**Pull request by yukim**

### CODE REVIEW

File: bin/cqlsh.py [hunk 1]
```diff
@@ -96,7 +96,7 @@
 # >>> webbrowser._tryorder
 # >>> webbrowser._browser
 #
-if len(webbrowser._tryorder) == 0:
+if webbrowser._tryorder is None or len(webbrowser._tryorder) == 0:
```
Comment (dineshjoshi) [inline bin/cqlsh.py:99]: Can you just call `webbrowser.get()` and leave the rest of the code as-is? This will initialize the `_tryorder` variable.

**Review by dineshjoshi — CHANGES_REQUESTED** [review]

File: bin/cqlsh.py (hunk 1 above)
Comment (nastra) [reply bin/cqlsh.py:99]: in a Docker testing environment it's likely that no default browser is installed and so you could end up in https://github.com/python/cpython/blob/3.7/Lib/webbrowser.py#L65, so I'm +1 with the fix here

Comment (yukim) [reply bin/cqlsh.py:99]: I tried `webbroser.get()` and ran in docker container, and I hit the error mentioned. So I think it is safer to check if it is `None`.

**Comment by jeffwidman** [comment]
Is this still relevant? Looks like the relevant code on `trunk` is a bit different than when this PR was opened...

If no longer relevant, can you please close this?

**Pull request closed without merging**